tests/client/api_restful/test_send_tagged_data.c

```diff
@@ -0,0 +1,148 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdio.h>
+#include <unity/unity.h>
```

Comment:
`unity.h` is not a system header. Move it to other project headers.

Reply:
Done.